projects/storefrontlib/src/cms-components/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.html

```diff
@@ -12,7 +12,7 @@
       <button
         type="button"
         class="close"
-        [attr.aria-label]="'common.close' | cxTranslate"
+        [attr.aria-label]="'addToCart.closeModal' | cxTranslate"
```

Comment:
Just a comment, we apply the same aria attribute in two templates in two different styles. 

- in `added-to-cart-dialog.component.html` we are using property binding `[attr.aria-label]="'addToCart.closeModal' | cxTranslate"`
- in `cart-item.component.html` we do string interppolation `attr.aria-label="{{ 'addToCart.removeFromCart' | cxTranslate }}"` 

is it possible to make it consistent across both templates?